test/fixtures/commits-with-labels.json

```diff
@@ -87,7 +87,7 @@
                     "edges": [
                       {
                         "node": {
-                          "mergeCommit": {"oid": "35abf13fa8acb3988aa086f3eb23f5ce1483cc5d"},
+                          "mergeCommit": {"oid": "fcd1c890dc1526f4d62ceedad561f498195c8939"},
```

Comment:
I switched the OID to an OID earlier in the list, so that we covered the edge-case addressed in this PR.